orderer/common/types/channelinfo.go

```diff
@@ -59,6 +59,7 @@ const (
 	StatusOnBoarding Status = "onboarding"
 	// The orderer is not storing any blocks for this channel.
 	StatusInactive Status = "inactive"
+	StatusFailure  Status = "failed"
```

Comment:
Add a comment here to explain the status? 

And I think we should have the constant name consistent with the value, so either StatusFailure = "failure" or StatusFailed = "failed". Not sure how I feel about "failed"... maybe "error" instead? Any thoughts @tock-ibm @jyellick ?

Reply:
For now, I went with StatusFailed. The comment you may hate.